src/Pages/OfficerDB/Edits/Editor.js

```diff
@@ -0,0 +1,157 @@
+import React, { useState } from 'react'
+import {
+  Modal,
+  ModalHeader,
+  ModalBody,
+  Dropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem,
+  Input
+} from 'reactstrap'
+import ConfirmModule from './ConfirmationModule'
+import Uploader, { upload, getFileName } from './Uploader'
+
+const enums = require('../../../Enums')
+const api = require('../../../APIFunctions/OfficerDB')
+
+export default function Editor (props) {
+  const [roleDropdown, setRoleDropdown] = useState(false)
+  const [role, setRole] = useState(props.officer.role)
+  const [linkedIn, setLinkedIn] = useState('')
+  const [github, setGithub] = useState('')
+  const [quote, setQuote] = useState('')
+  const [picture] = useState(props.officer.pictureName)
+  const optionalInputs = [
+    {
+      placeholder: 'New linkedIn: ' + props.officer.linkedIn,
+      onChange: event => setLinkedIn(event.target.value)
+    },
+    {
+      placeholder: 'New github: ' + props.officer.github,
+      onChange: event => setGithub(event.target.value)
+    },
+    {
+      placeholder: 'New quote: ' + props.officer.quote,
+      onChange: event => setQuote(event.target.value)
+    }
+  ]
+
+  async function submitOfficer () {
```

Comment:
this function should go inside `APIFunctions/`

Reply:
actually, that's my bad I renamed it into ```handleSubmitOfficer``` its not an axios function so i wouldn't put it in APIfunctions